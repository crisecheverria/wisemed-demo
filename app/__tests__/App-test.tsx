import * as React from "react";
import { render, screen } from "@testing-library/react-native";

import RootLayout from "../_layout";

describe(`<RootLayout />`, () => {
  test(`renders header text`, () => {
    render(<RootLayout />);

    expect(screen.getByText(/Traumatolog√≠a/i)).toBeTruthy();
  });
});
