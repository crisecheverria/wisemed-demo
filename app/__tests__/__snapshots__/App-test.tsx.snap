// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RootLayout /> renders correctly 1`] = `
<RNCSafeAreaView
  edges={
    {
      "bottom": "additive",
      "left": "additive",
      "right": "additive",
      "top": "additive",
    }
  }
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#F3EDF7",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      {
        "alignSelf": "center",
        "backgroundColor": "#fff",
        "borderRadius": 5,
        "elevation": 5,
        "height": 367,
        "shadowColor": "#000",
        "shadowOffset": {
          "height": 2,
          "width": 0,
        },
        "shadowOpacity": 0.2,
        "shadowRadius": 4,
        "width": 276,
      }
    }
  >
    <View
      style={
        {
          "backgroundColor": "#154FBF",
          "borderTopLeftRadius": 5,
          "borderTopRightRadius": 5,
          "flexDirection": "row",
          "justifyContent": "space-between",
          "padding": 12,
        }
      }
    >
      <View>
        <Text
          style={
            {
              "color": "#fff",
              "fontSize": 16,
              "fontWeight": 600,
              "lineHeight": 24,
            }
          }
        >
          Traumatología
        </Text>
        <Text
          style={
            {
              "color": "#fff",
              "fontSize": 12,
              "fontWeight": 600,
              "lineHeight": 18,
            }
          }
        >
          Dr. José Pedro Sans
        </Text>
      </View>
      <Image
        source={
          {
            "testUri": "../../../assets/images/doctor-icon.png",
          }
        }
        style={
          {
            "height": 50,
            "marginTop": 2,
            "width": 50,
          }
        }
      />
    </View>
    <View
      style={
        {
          "flexDirection": "row",
          "marginTop": 16,
        }
      }
    >
      <Image
        source={
          {
            "testUri": "../../../assets/images/face-24.png",
          }
        }
        style={
          {
            "height": 24,
            "marginLeft": 16,
            "width": 24,
          }
        }
      />
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "fontWeight": 500,
            "lineHeight": 21,
          }
        }
      >
        Jorge Avendaño Pérez 35 anos
      </Text>
    </View>
    <View
      style={
        {
          "padding": 16,
        }
      }
    >
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Ficha médica:
        </Text>
         77884
      </Text>
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Diagnóstico:
        </Text>
         Calcificación Talón
      </Text>
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Intervención
        </Text>
         Extirpación en talón.
      </Text>
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Evaluación preantestésica:
        </Text>
        Sí
      </Text>
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Tiempo de solicitud:
        </Text>
        3 dias
      </Text>
      <Text
        style={
          {
            "color": "#000",
            "fontSize": 14,
            "marginVertical": 2,
          }
        }
      >
        <Text
          style={
            {
              "color": "#154FBF",
              "fontSize": 12,
              "fontWeight": "bold",
              "lineHeight": 16,
            }
          }
        >
          Suspensiones:
        </Text>
        2
      </Text>
    </View>
    <View
      style={
        {
          "flexDirection": "row",
        }
      }
    >
      <Image
        source={
          {
            "testUri": "../../../assets/images/icon-cardio.png",
          }
        }
        style={
          {
            "height": 24,
            "marginLeft": 16,
            "width": 24,
          }
        }
      />
      <Image
        source={
          {
            "testUri": "../../../assets/images/icon-anestesia.png",
          }
        }
        style={
          {
            "height": 24,
            "marginLeft": 16,
            "width": 24,
          }
        }
      />
    </View>
    <View
      style={
        {
          "marginHorizontal": 16,
          "marginTop": 8,
          "paddingTop": 8,
        }
      }
    >
      <View
        style={
          [
            {
              "width": "100%",
            },
            {
              "zIndex": 5000,
            },
            {},
          ]
        }
      >
        <View
          accessibilityState={
            {
              "busy": undefined,
              "checked": undefined,
              "disabled": false,
              "expanded": undefined,
              "selected": undefined,
            }
          }
          accessibilityValue={
            {
              "max": undefined,
              "min": undefined,
              "now": undefined,
              "text": undefined,
            }
          }
          accessible={true}
          collapsable={false}
          focusable={true}
          onClick={[Function]}
          onLayout={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            {
              "alignItems": "center",
              "backgroundColor": "#fff",
              "borderColor": "#000",
              "borderRadius": 8,
              "borderWidth": 1,
              "flexDirection": "row",
              "justifyContent": "space-between",
              "minHeight": 50,
              "opacity": 1,
              "paddingHorizontal": 10,
              "paddingVertical": 3,
              "width": "100%",
              "zIndex": 5000,
            }
          }
        >
          <Text
            style={
              [
                {
                  "color": "#000",
                  "flex": 1,
                },
                {},
                false,
                {},
              ]
            }
          >
            Select an item
          </Text>
          <View
            style={
              [
                {
                  "marginLeft": 10,
                },
                {},
              ]
            }
          >
            <Image
              source={
                {
                  "testUri": "../../react-native-dropdown-picker/src/themes/light/icons/arrow-down.png",
                }
              }
              style={
                [
                  {
                    "height": 20,
                    "width": 20,
                  },
                  {},
                ]
              }
            />
          </View>
        </View>
      </View>
    </View>
  </View>
</RNCSafeAreaView>
`;
